src/hooks/useBuyout/index.ts

```diff
@@ -55,11 +55,42 @@ function handleBuyoutError(error: string) {
 
 export const useBuyout = (): BuyoutSettings => {
   const { api } = useNodeInfoState().state;
-  const { tenantName } = useGlobalState();
   const { walletAccount } = useGlobalState();
   const [minimumBuyout, setMinimumBuyout] = useState<number>(0);
   const [maximumBuyout, setMaximumBuyout] = useState<number>(0);
   const [sellFee, setSellFee] = useState<PerMill>(new PerMill(0));
+  const { getNativeAssetMetadata, getAssetMetadata } = useAssetRegistryMetadata();
+
+  const [nativeCurrency, setNativeCurrency] = useState<OrmlTraitsAssetRegistryAssetMetadata | undefined>(undefined);
+  const [currencies, setCurrencies] = useState<OrmlTraitsAssetRegistryAssetMetadata[]>([]);
+
+  useEffect(() => {
+    async function fetchAllowedCurrencies() {
+      const allowedCurrencies = [];
+      if (api) {
+        const allowedCurrenciesEntries = await api.query.treasuryBuyoutExtension.allowedCurrencies.entries();
+
+        for (const currency of allowedCurrenciesEntries) {
+          if (currency.length && currency[0] && currency[0].toHuman()) {
+            const currencyXCMId: { XCM: number } = (currency[0].toHuman() as { XCM: number }[])[0];
```

Comment:
This is a bit ugly and we should try to use proper types here. We first need to update them in the pendulum.js package though, so I'm fine if we keep it like this for now.